app/components/app_consent_component.rb

```diff
@@ -14,7 +14,7 @@ def initialize(patient_session:, programme:)
 
   def consents
     @consents ||=
-      patient.consent_outcome.all[programme].sort_by(&:created_at).reverse
+      patient.consents.includes(:consent_form, :parent).order(created_at: :desc)
```

Comment:
Does this need to be scoped to `programme`?

Reply:
Yes, it does, I've amended the commit.